content/docs/security/hipaa.md

```diff
@@ -66,6 +66,129 @@ HIPAA is a federal law that sets national standards for the protection of health
+| 18                 | SESSION                                                          | Log message level. |
+| 19                 | 1                                                                | Session ID. |
+| 20                 | 1                                                                | Subsession or transaction ID. |
+| 21                 | DDL                                                              | Statement type: Data Definition Language. |
+| 22                 | CREATE SCHEMA                                                    | Statement tag/type. |
+| 23–26              | *(empty)*                                                        | Reserved/unused fields. |
+| 27                 | CREATE SCHEMA IF NOT EXISTS healthcare                           | Full SQL text of the statement. |
+| 28                 | `<not logged>`                                                   | Parameter values (redacted or disabled by settings like `pgaudit.log_parameter`). |
+| 29–35              | *(empty)*                                                        | Reserved/unused fields. |
+| 36                 | neon-internal-sql-editor                                         | Application name or source of the query (e.g., SQL Editor in the Neon Console). |
+
+### Extension configuration
+
+The `pgaudit` extension is preloaded on HIPAA-enabled Neon projects. For extension version information, see [Supported Postgres extensions](/docs/extensions/pg-extensions).
+
+### Console operation logging
+
+Neon logs operations performed via the Neon Console interface. These actions are initiated through the UI and correspond to API requests made to the Neon backend. Examples of logged operations include:
+
+- **Project management**: creating, deleting, renaming projects
+- **Branch management**: creating, deleting, renaming branches; restoring from backups
+- **Compute management**: starting, stopping, scaling compute instances
+- **Database and role management**: creating or deleting databases; resetting Postgres role passwords.
+- **Connection setup**: viewing or copying connection strings.
+- **Organization and access**: inviting users, assigning roles, or removing users from an organization.
+
+To protect sensitive information, Neon filters data in audit logs using the following approach:
+
+- Sensitive fields (such as `connection_uri` and `password`) are excluded from logs. These are identified using `x-sensitive` tags in the OpenAPI specification.
+- `GET` requests: Only query parameters are logged; response payloads are not recorded.
+- Mutation requests (`PATCH`, `PUT`, `POST`, `DELETE`): Request and response bodies are logged with sensitive fields redacted.
+
+This logging approach ensures that all meaningful Neon Console activity is auditable while safeguarding user credentials and other sensitive data.
+
```

Comment:
@adi-griever I'd like to add a console operation log event example. How do I find one?

Reply:
@danieltprice see FAQ: https://www.notion.so/neondatabase/Neon-Internal-HIPAA-FAQ-1b9f189e004780bfacf4d9bbe5cd0d0a?source=copy_link#207f189e004780e49f56c78389501749